### PR TITLE
feat: created badges with different types

### PR DESCRIPTION
**Pull request by amarphule**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,6 +92,45 @@ <h1>Avatar</h1>
...
...Ci%2520class%253D%2522far%2520fa-heart%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-icon-val%2522%253E3%253C%252Fspan%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-wrapper%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cbutton%253EAction%253Cspan%2520class%253D%2522badge-icon-val%2522%253E3%253C%252Fspan%253E%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-wrapper%2520avatar-l%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-img%2522%253E3%253C%252Fspan%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-wrapper%2520avatar-l%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-stat%2522%253E%253C%252Fspan%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E"
```
Comment (tanishk15sharma) [inline index.html:120]: tip : you can use width:100% to make it more responsive

**Review by tanishk15sharma — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -91,6 +92,45 @@ <h1>Avatar</h1>
...
+              style="
+                width: 714px;
```
Comment (tanishk15sharma) [inline index.html]: looks to goot to me you can merge it .
tip : you can change `iframe` width : 100%  to make it more responsive .

**Review by tanishk15sharma — APPROVED** [review]

File: components/badges.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  padding: 0.5rem;
+  font-weight: 700;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  border: 4px solid var(--hover-bg);
+  color: var(--hover-bg);
+  background-color: var(--success-color);
+}
```
Comment (horsemaker) [inline components/badges.css:46]: I see many properties are repeated, can't anything be done about that? Can you make a common class to do that?

File: index.html (hunk 2 above)
Comment (amarphule) [reply index.html]: @tanishk15sharma  Thank you for the suggestion. made changes.

File: components/badges.css (hunk 3 above)
Comment (amarphule) [reply components/badges.css:46]: @horsemaker thank you for pointing it out. made common class for similar property.

**Pull request merged**